src/shared/lib/comparison/ComparisonStore.ts

```diff
@@ -2014,6 +2037,120 @@ export default abstract class ComparisonStore {
         });
     }
 
+    readonly molecularProfilesInStudies = remoteData<MolecularProfile[]>(
```

Comment:
there's already molecularProfilesInActiveStudies.   I wonder whether that would be sufficient here?

Reply:
I guess so. I will test this.